io.github.lainsce.Quilter.json

```diff
@@ -0,0 +1,106 @@
+{
+    "app-id" : "io.github.lainsce.Quilter",
+    "runtime" : "org.gnome.Sdk",
+    "runtime-version" : "3.38",
+    "sdk" : "org.gnome.Sdk",
+    "command" : "io.github.lainsce.Quilter",
+    "rename-icon" : "io.github.lainsce.Quilter",
+    "finish-args" : [
+        "--filesystem=xdg-data/io.github.lainsce.Quilter/",
```

Comment:
Your application has write here by default, just needs to follow `$XDG_DATA_HOME`.

Reply:
so just `--filesystem=xdg-data` instead?